src/App.vue

```diff
@@ -25,6 +26,18 @@
 			Footer,
 			Navigation,
 			Carousel
+		},
+		data() {
+			return {
+				json: {},
```

Comment:
why json and not a structure, you name variables not by type, but by expected content